<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newPurchaseDialog</class>
 <widget class="QDialog" name="newPurchaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelItemInStock">
      <property name="text">
       <string>Items In Stock</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCustomerName">
      <property name="text">
       <string>Customer Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelQuantity">
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBoxItemInStock">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCustomerName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxQuantity">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAddPuchase">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Purchase</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCheckOut">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Out</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
